src/scss/survey.scss

```diff
@@ -0,0 +1,62 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import 'includes/lib';
+
+.survey-banner {
+    @include gradient-horizontal($start-color: #ff00ff, $end-color: #2e2aa6);
+    bottom: 0;
+    color: #fff;
+    display: none;
+    left: 0;
+    padding-bottom: 10px;
```

Comment:
Totally a small thing, but if we make this bottom padding 60px and position `bottom: -50px;` there will be some extra hidden banner so we won't get that brief glimpse of the page behind it when the banner pops up.